karma.base.js

```diff
@@ -21,6 +21,7 @@ module.exports = {
     // and sets up test helpers (chai-enzyme, etc.)
     path.join(__dirname, 'node_modules/whatwg-fetch/fetch.js'),
     path.join(__dirname, 'test/index.js'),
+    'test/fixtures/**/*',
```

Comment:
Yeah, I don't think `path.join` is necessary, I just opted out of using it because `**/*` is a glob and not actually a path so it seemed weird to use it.